components/scalr/sources/new-run-completed/new-run-completed.mjs

```diff
@@ -0,0 +1,28 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "New Run Completed (Instant)",
+  version: "0.0.1",
+  key: "scalr-new-run-completed",
+  description: "Emit new event on each new completed run.",
```

Comment:
I like to put in doc links if possible, especially when there aren't any other components with links to the docs.
```suggestion
  description: "Emit new event on each new completed run. [See the documentation](https://docs.scalr.io/reference/create_webhook_integration)",
```